as-vending/Dockerfile

```diff
@@ -28,13 +29,13 @@ COPY . .
 RUN make gobuild
 
 # Next image - Copy built Go binary into new workspace
-FROM alpine
+FROM alpine:3.17
 
 LABEL license='SPDX-License-Identifier: BSD-3-Clause' \
-  copyright='Copyright (c) 2022: Intel'
+  copyright='Copyright (c) 2023: Intel'
 
 RUN apk --no-cache add zeromq
```

Comment:
do we need zeromq?

```suggestion

```

Reply:
Good catch. That should be removed also.